example/cross_validation_for_iris.py

```diff
@@ -36,6 +33,6 @@
     gb_score += gb.score(xs_test, y_test)
 
 rgf_score /= n_folds
-print('RGF Classfier score: {0}'.format(rgf_score))
+print('RGF Classfier score: {0:.5f}'.format(rgf_score))
 gb_score /= n_folds
-print('Gradient Boosting Classfier score: {0}'.format(gb_score))
+print('Gradient Boosting Classfier score: {0:.5f}'.format(gb_score))
```

Comment:
Sorry. I'd made this comment before you guided me for to contribute properly. I suppose to add more examples. And this commit is unnecessary here.